### ISSUE TITLE
Content scripts are not run in dev mode until they are changed in Chrome

### ISSUE THREAD

**Issue by renataogarcia** [wxt-dev/wxt#1252]
### Describe the bug

The content scripts are not registered on initial load. Once they are changed they are run.

It works on Firefox but not on Chrome.

The logs from the service worker doesn't display the message the scripts have been reloaded right after 
start.
![image](https://github.com/user-attachments/assets/7104e883-52b1-472a-97d2-826e10cf28ae)
[console-1733436443448.log](https://github.com/user-attachments/files/18030252/console-1733436443448.log)

### Reproduction

Followed steps from the discussion from the start guide.

### Steps to reproduce

pnpm dlx wxt@latest init .
pnpm i
pnpm dev

Open google.com on the browser and no console message is shown from content-script.

Also the steps from the discussion to check the console from the discussion below doesn't have the same results.
https://github.com/wxt-dev/wxt/discussions/1248#discussioncomment-11469046

### System Info

```shell
System:
    OS: Linux 6.12 Arch Linux
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i9-11900 @ 2.50GHz
    Memory: 18.71 GB / 31.10 GB
    Container: Yes
    Shell: 5.2.37 - /bin/bash
  Binaries:
    Node: 20.14.0 - ~/.nvm/versions/node/v20.14.0/bin/node
    npm: 10.7.0 - ~/.nvm/versions/node/v20.14.0/bin/npm
    pnpm: 8.15.4 - ~/.nvm/versions/node/v20.14.0/bin/pnpm
  Browsers:
    Chromium: 131.0.6778.85
```

### Used Package Manager

pnpm

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/wxt-dev/wxt/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://wxt.dev/guide/installation.html).
- [X] Check that there isn't [already an issue](https://github.com/wxt-dev/wxt/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/wxt-dev/wxt/discussions) or join our [Discord Chat Server](https://discord.gg/ZFsZqGery9).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by aklinker1** [comment]
Yeah, unfortunately, I tested these exact reproduction steps in the discussion on Mac, couldn't reproduce. I'll give it a go on arch as well

**Comment by aklinker1** [comment]
Fixed by https://github.com/wxt-dev/wxt/pull/1253. @renataogarcia can you confirm by trying out the fix?

```sh
pnpm i https://pkg.pr.new/wxt@1253
```

If it works, I'll merge the PR and do a release.

**Comment by renataogarcia** [comment]
@aklinker1 Yep, it works! Amazing turnaround, thanks a lot!

**Comment by aklinker1** [comment]
Released in [`v0.19.19](https://github.com/wxt-dev/wxt/releases/tag/wxt-v0.19.19)
